third_party/rust/curve25519_dalek/v3/BUILD.gn

```diff
@@ -1,97 +1,103 @@
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/lib.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/macros.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/montgomery.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/prelude.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/ristretto.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/scalar.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/traits.rs",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/window.rs",
   ]
   inputs = [
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/CHANGELOG.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/CODE_OF_CONDUCT.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/CONTRIBUTING.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/README.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/docs/avx2-notes.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/docs/ifma-notes.md",
-    "//brave/third_party/rust/curve25519_dalek/v3/crate/docs/parallel-formulas.md",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/../README.md",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/../docs/avx2-notes.md",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/../docs/ifma-notes.md",
+    "//brave/third_party/rust/chromium_crates_io/vendor/curve25519-dalek-3.2.0/src/../docs/parallel-formulas.md",
   ]
 
-  # Unit tests skipped. Generate with --with-tests to include them.
   build_native_rust_unit_tests = false
   edition = "2015"
-  cargo_pkg_version = "3.2.1"
+  cargo_pkg_version = "3.2.0"
   cargo_pkg_authors = "Isis Lovecruft <isis@patternsinthevoid.net>, Henry de Valence <hdevalence@hdevalence.ca>"
   cargo_pkg_name = "curve25519-dalek"
   cargo_pkg_description = "A pure-Rust implementation of group operations on ristretto255 and Curve25519"
   library_configs -= [ "//build/config/compiler:chromium_code" ]
   library_configs += [ "//build/config/compiler:no_chromium_code" ]
   executable_configs -= [ "//build/config/compiler:chromium_code" ]
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
+  proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
+  proc_macro_configs += [ "//build/config/compiler:no_chromium_code" ]
   deps = [
     "//brave/third_party/rust/byteorder/v1:lib",
     "//brave/third_party/rust/digest/v0_9:lib",
     "//brave/third_party/rust/rand_core/v0_5:lib",
     "//brave/third_party/rust/subtle/v2:lib",
     "//brave/third_party/rust/zeroize/v1:lib",
-    "//third_party/rust/serde/v1:lib",
   ]
   features = [
     "alloc",
-    "serde",
```

Comment:
double-checked this removal and it appears to be correct. This appears to only be enabled on v4